### ISSUE TITLE
Allow to deactivate all visible tracks at once

### ISSUE THREAD

**Issue by scaidermern** [osmandapp/OsmAnd#13639]
If you have many tracks enabled it is tedious to disable them again.

Enabling multiple tracks at once is very easy. Go to My Places -> Tracks and tap on the map icon at the bottom. Now it is possible to enable whole directories of tracks.

Disabling multiple tracks at once is time-consuming. Using the same technique as above is not possible. The "Visible" category has no check box to disable all visible tracks at once.

Example:
![photo_2022-01-28_10-39-48](https://user-images.githubusercontent.com/1040784/151523648-be0fd67a-742a-4c08-adb3-e523d7fb19dc.jpg)

- Regular folders like "Bicycle/Spreewald" have a checkbox to enable all tracks at once.
- "Visible" lacks such a checkbox. Each track has to be disabled individually.

**Comment by pebogufi** [comment]
It IS possible at least if only one track group (subdir) is affected (as a workaround).
Use the checkbox left of "Bicycle/Spreewald" (or the apropirate group) then check ON, (now are all on) then check again for OFF (now are all off) then check in top part "auf der Karte anzeigen". If more groups are affected, repeat with all. 
Of course a "uncheck all" in "visible" would be better.
Different solution:
"Configure map", then "tracks", move slider to off.

**Comment by scaidermern** [comment]
> It IS possible at least if only one track group (subdir) is affected (as a workaround). Use the checkbox left of "Bicycle/Spreewald" (or the apropirate group) then check ON, (now are all on) then check again for OFF (now are all off) then check in top part "auf der Karte anzeigen".

Thanks, this works. I would have never expected that. Not intuitive at all :)

**Comment by sonora** [comment]
A good UI is how many file managers have it:
![Screenshot_20240618-074526_My Files](https://github.com/osmandapp/OsmAnd/assets/845735/67b5baa8-770a-49e1-bdc0-4e318adca4f6)

**Comment by vshcherb** [comment]
You can deselect all tracks now in Configure Map screen

**Comment by sonora** [comment]
> You can deselect all tracks now in Configure Map screen

That's true. I was addressing how to perhaps more discoverably y deselect all tracks in a folder.
